projects/netgrif-components-core/src/lib/data-fields/i18n-field/language-icons.service.ts

```diff
@@ -0,0 +1,255 @@
-.125.129v-.023a.11.11 0 0 0 .094-.051c.008-.02.012-.035.012-.055h.02c0 .039-.02.086-.059.109-.02.012-.043.02-.066.02zm-.129-.129h.02c0 .039.02.082.059.098.016.008.031.012.051.012v.02c-.047 0-.086-.02-.109-.059-.012-.02-.02-.043-.02-.066zm.129-.129v.02c-.043 0-.082.023-.098.059a.12.12 0 0 0-.012.051h-.02c0-.043.02-.086.059-.109.02-.012.043-.02.07-.02zm0 0" class="sr-B"/><path d="M7.688 4.055c.066-.004.047.117.039.145-.02.039-.035.039-.062.004-.016-.023-.047-.145.023-.148zm0 0" class="sr-D"/><path d="M7.73 4.203l-.02-.008h.004v-.012l.004-.008v-.008l.004-.008v-.066h-.004v-.008l-.004-.004v-.004l-.004-.004h-.004l-.004-.004h-.004l-.004-.004h-.008v-.02h.016l.004.004h.008l.004.004v.004l.008.004.004.004.004.008v.004l.004.004v.016l.004.008v.074l-.004.004v.012l-.004.004v.012H7.73zm-.074.008l.02-.016.004.008v.004h.004v.004h.004v.004h.012l.004-.004v-.004h.004v-.004h.004v-.008l.02.008v.004l-.004.004v.004h-.004v.004l-.008.008h-.008v.004H7.68v-.004h-.004v-.004h-.004v-.004h-.004v-.004h-.004v-.008h-.008zm.027-.168v.02h-.008v.004h-.008v.004H7.66v.004l-.004.004v.008h-.004v.066l.004.004v.012l.004.004v.008l.004.004v.004l.008.004v.004h.004l-.023.012v-.008l-.004-.004v-.004l-.004-.008V4.18l-.004-.008v-.004l-.004-.004v-.016l-.004-.008v-.059l.004-.008V4.07l.016-.016h.004l.004-.008.004-.004h.012l.004-.004zm0 0" class="sr-B"/><path d="M7.691 4.121c.16-.012.117.277.086.344-.039.082-.078.086-.133.008-.039-.059-.117-.34.043-.352zm0 0" class="sr-C"/><path d="M7.789 4.469l-.02-.008.008-.016v-.008l.004-.012.004-.008v-.012l.004-.012.004-.016v-.027l.004-.012v-.102l-.004-.016v-.012l-.008-.016-.004-.012-.008-.008-.004-.012-.008-.008-.004-.008-.012-.008-.008-.004-.012-.004-.008-.004h-.027v-.02h.027l.012.004.016.004.023.016.016.016.008.016.004.012v.016l.008.012.004.012v.016l.004.016v.125l-.004.016v.016l-.004.012v.012l-.008.012v.012l-.004.008v.008l-.008.008zm-.152.008l.016-.008.004.004.004.008.008.004.004.008h.004V4.5h.008l.008.008h.008l.004.004h.02v-.004h.004l.004-.004V4.5l.008-.008h.004l.004-.008v-.008l.008-.004.008-.008.004-.008.02.008-.008.008-.004.012-.004.004-.008.016-.008.008-.004.008-.004.004-.008.004h-.004v.008h-.008l-.004.004h-.023l-.008-.004h-.004l-.004-.008v-.004l-.008-.004-.004-.004-.008-.004-.012-.012-.008-.016zm.051-.344v-.02.02h-.012l-.016.004-.008.004-.012.004-.023.023-.004.008-.008.012-.004.012v.012l-.004.008v.016l-.004.016v.012l.004.012v.027l.004.016v.016l.008.016v.012l.004.012.004.016.004.008.004.012.004.016.008.016.004.012.004.008.004.004-.02.012v-.008l-.008-.016-.004-.012v-.012l-.008-.008-.004-.016-.008-.023-.004-.016v-.016l-.008-.012V4.32l-.004-.012v-.035l-.004-.016v-.016l.004-.016v-.012l.004-.012v-.016l.008-.012.004-.012.016-.023.008-.008.023-.016.012-.004.016-.004h.016zm0-.02v.02zm0 0" class="sr-B"/><path d="M6.395 4.227c.066-.02.078.102.07.133-.008.043-.023.047-.055.023-.02-.027-.078-.133-.02-.156zm0 0" class="sr-D"/><path d="M6.477 4.363l-.02-.004v-.059l-.004-.004v-.016l-.004-.004V4.27l-.004-.004v-.004l-.008-.004-.004-.004-.004-.008-.008-.004h-.008l-.004-.004h
```

Comment:
can we make domSanitizer protected ? for future extendation from another libraries